docs/api-and-integration/octopus.server.exe-command-line/index.md

```diff
@@ -6,32 +6,34 @@ position: 90
 
 **Octopus.Server.exe** is the executable that runs the Octopus Server instance. It includes many helpful commands that allow you to manage the instance, including; authentication, configuration, diagnostics and running the service.
 
-## Commands {#Octopus.Server.exeCommandLine-Commands}
+## Commands {#octopus.server.exeCommandLine-Commands}
 
-`Octopus.Server.exe` supports the following commands:
+`octopus.server.exe` supports the following commands:
 
 - **`admin`**:  Reset admin user passwords, re-enable them, and ensure they are in the admin group.
-- **`builtin-worker`**:  Configure the built-in worker used to run deployment actions and scripts on the Octopus Server.  Available in **Octopus 2018.1.0** and later.
```

Comment:
I _think_ this is okay to remove. I wonder if we should put a footer (or similar) on these pages to show what version they apply to?

Reply:
Chatted to @wordlee and he's okay with it. We've got LegacyDocs to handle that kind of thing.